plugins/palo_alto_pan_os/plugin.spec.yaml

```diff
@@ -949,7 +949,7 @@ actions:
         required: true
   get_addresses_from_group:
     title: Get Addresses from Group
-    description: Get addresses from an address group
+    description: Get addresses from an address group. This action works with a direct connection to the firewall
```

Comment:
For each action, please rephrase and punctuate the description. Please replace `works with` with `uses`, and add `.` to the end of each sentence.